packages/data-seeder/src/locations.ts

```diff
@@ -71,90 +65,6 @@ function validateAdminStructure(locations: TypeOf<typeof LocationSchema>) {
-  locationNodeMap.set('0', { id: '0', children: [] })
-  locations.forEach((loc) => {
-    const parent = locationNodeMap.get(loc.partOf.split('/')[1])
-    if (!parent) {
-      raise(`Parent location "${loc.partOf}" not found for ${loc.name}`)
-    }
-    parent.children.push(loc.id)
-  })
-
-  function validateTree(root: string) {
-    const node = locationNodeMap.get(root)!
-    const location = locationsMap.get(node.id)!
-    const childLocations = node.children.map(
-      (child) => locationsMap.get(child)!
-    )
-    const accumulatedStatistics = new Map<
-      number,
-      { male_population: number; female_population: number; population: number }
-    >()
-    for (const [year, parentStats] of location.statistics.entries()) {
-      if (
-        parentStats.population <
-        parentStats.male_population + parentStats.female_population
-      ) {
-        statisticsErrors.push(
-          new Error(
-            `Location: ${
-              location.name
-            }, year: ${year} -> Sum of male population and female population ${
-              parentStats.male_population + parentStats.female_population
-            } is higher than the total population ${parentStats.population}`
-          )
-        )
-      }
-      childLocations.forEach((childLocation) => {
-        if (!childLocation.statistics.get(year)) {
-          statisticsErrors.push(
-            new Error(
-              `Couldn't find child location data: ${childLocation.name}, year: ${year} -> but that year's data exists for parent: ${location.name}`
-            )
-          )
-        }
-        const currentStats = accumulatedStatistics.get(year) ?? {
-          male_population: 0,
-          female_population: 0,
-          population: 0
-        }
-        const childStats = childLocation.statistics.get(year) ?? {
-          male_population: 0,
-          female_population: 0,
-          population: 0
-        }
-
-        accumulatedStatistics.set(year, {
-          male_population:
-            currentStats.male_population + childStats.male_population,
-          female_population:
-            currentStats.female_population + childStats.female_population,
-          population: currentStats.population + childStats.population
-        })
-      })
-      if (childLocations.length > 0) {
-        const sumPopulation = accumulatedStatistics.get(year)!.population
-        if (sumPopulation > parentStats.population) {
-          statisticsErrors.push(
-            new Error(
-              `Location: ${location.name}, year: ${year} -> Sum of child locations' population ${sumPopulation} is greater than the total population ${parentStats.population}, The total population should be >= ${sumPopulation}`
-            )
-          )
-        }
-      }
-    }
-  }
-  locationNodeMap.get('0')!.children.forEach((childId) => validateTree(childId))
-  if (statisticsErrors.length > 0) {
-    raise(statisticsErrors.map((error) => error.message).join('\n'))
-  }
```

Comment:
@bvenceslas Correct me if I am wrong but the goal was to remove the requirement for all lower level locations to have statistics data but we still want to ensure that the top level locations i.e. the states have statistics data. But the current change removes any and all statistics validations